stdlib/@tests/stubtest_allowlists/common.txt

```diff
@@ -13,6 +13,10 @@ _collections_abc.ItemsView.__reversed__
 _collections_abc.KeysView.__reversed__
 _collections_abc.ValuesView.__reversed__
 _ctypes.CFuncPtr  # stubtest erroneously thinks it can't be subclassed
+_frozen_importlib_external.ExtensionFileLoader.get_filename  # Wrapped with _check_name decorator which changes runtime signature
+_frozen_importlib_external.FileLoader.get_filename  # Wrapped with _check_name decorator which changes runtime signature
+_frozen_importlib_external.FileLoader.get_resource_reader  # Wrapped with _check_name decorator which changes runtime signature
+_frozen_importlib_external.FileLoader.load_module  # Wrapped with _check_name decorator which changes runtime signature
```

Comment:
You put that in the "should be fixed" section, but if I understand your comments correctly, this isn't fixable?

Reply:
These are duplicates of allowlist lines already present in this section, they're just showing up in other modules. Neither the mismatch or the comment is new to this MR.

I believe they could be fixed; right now the stubs match the functions as written directly instead of their modified signature and I'm not totally sure why it was done that way, but I left it alone.